src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -67,6 +80,15 @@ export const InitializeClient: FC<OwnProps> = ({
     () => allPermissionTypes.sort((a, b) => collator.compare(a, b)),
     [allPermissionTypes]
   )
+  const {bucket} = useContext(WriteDataDetailsContext)
+
+  useEffect(() => {
+    dispatch(getBuckets())
+  }, [])
```

Comment:
I've been [persuaded](https://github.com/facebook/react/issues/14920) that it's actually a good idea to add even the functions that shouldn't change (e.g. `dispatch`) to the dependency array. Since `dispatch` shouldn't change it's safe to add (and if dispatch did change, that's bad and we probably should re-run the hook). Otherwise, I bet we're adding another linter warning to our warning collection here.